src/resources/information/en.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>About Thorium Reader</title>
+  </head>
+<body style="font-family: 'Gill Sans', sans-serif">
+  <h1>About Thorium Reader</h1>
+
+  <h2>Introduction</h2>
+
+  <p>Thorium Reader is the Reading Application of choice for Windows 10, MacOS and Linux.
+  <p>This <a href="https://edrlab.org">EDRLab</a> application is in constant development and aims at becoming a reference for accessing EPUB publications , PDF documents, audiobooks and visual narratives, LCP protected or not.
+  <p>It is localized in a large set of languages, each version offering new languages.
+  <p>Huge efforts are also made to get Thorium Reader highly accessible for visually impaired and dyslexic people.
+  <p>This application is free, with no ads and no leaks of private data.
+  <p>This is the perfect tools for heavy readers, library patrons and students.
+  
+  <h2>Supported publication formats</h2>
+
+  <p>EPUB 2: this is a legacy format, only adequate for basic ebooks. Textual content is formatted in XHTML 1 and no multimedia feature is supported. EPUB 2 ebooks are still accessible if properly formatted.
+  <p>EPUB 3: this is the recommended format for the 21st century. Textual content is formatted in (X)HTML 5, it is possible to create fixed-layout ebooks, enriched ebooks and accessibiltiy features are well designed. 
+  <p>PDF: despite a well-known lack of flexibility (no reflow option) and accessibility, many ebooks are still available in PDF only, especially in public libraries.
+  <p>audiobooks: a <a href="https://www.w3.org/TR/audiobooks/">standardized format for audiobooks</a> has been designed by the W3C Publishing Working Group in 2020, and EDRLab was part of this WG. Thorium is supporting the <a href="https://www.w3.org/TR/lpf/">packaged (LPF)<a> audiobook format, which will soon be widely used by booksellers, or so we hope.  
+  <p>visual narratives: EDRLab has designed a standard format for comics, manga, webtoons and other styles of digital works based on sequances of images. <a href="https://www.edrlab.org/open-standards/">Divina</a> is the name of this format. This is still an experimental format, and we hope that publishers will size it in 2021+. 
+
+  <h2>Accessibility</h2>
+
+  <p>EDRLab has strong relationships with the <a href="https://daisy.org/">DAISY Consortium<a>, and we want to make Thorium Reader one on the most accessible reading application on the market; better than most commercial products. 
```

Comment:
Incorrect closing `a` tag:
`<a href="https://daisy.org/">DAISY Consortium<a>`

The DOM is broken, causing problems in the "search" feature (probably other CSS selector related problems too)
 
See: https://github.com/edrlab/thorium-reader/issues/1304

Reply:
Also `<a href="https://www.w3.org/TR/lpf/">packaged (LPF)<a>`

Reply:
@llemeurfr can you please make sure the XHTML file is valid?

https://validator.w3.org/nu/?doc=https%3A%2F%2Fraw.githack.com%2Fedrlab%2Fthorium-reader%2F6e255f5b3cbd6dfd2627d1277cd933b449342bf7%2Fsrc%2Fresources%2Finformation%2Fen.html

Reply:
Laurent, I have done this:
https://github.com/edrlab/thorium-reader/pull/1199/commits/530c537a91d1cf82b4695ef6af78f17b939091b1

Reply:
The XHTML document is now valid: https://validator.w3.org/nu/?doc=https%3A%2F%2Fraw.githack.com%2Fedrlab%2Fthorium-reader%2F530c537a91d1cf82b4695ef6af78f17b939091b1%2Fsrc%2Fresources%2Finformation%2Fen.html